index.ts

```diff
@@ -0,0 +1,10 @@
+import { IncomingMessage, ServerResponse } from 'http'
+
+type NextFunction = (err?: any) => void
+
+export = function ienoopen () {
```

Comment:
I couldn't figure out a way to maintain backwards compatibility and use "normal" exports. In other words, is there a better way to make this work?

```js
// JavaScript, not TypeScript
console.log(typeof require('ienoopen')) // 'function'
```

Reply:
`module.exports =` has no equivalent in "normal" ES6 exports. `export =` is the Typescript equivalent of `module.exports =`, so you got it right.